01_fib_series/fibseries.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func FibonacciRecursion(n int) int {
```

Comment:
Since the FibonacciRecursion is called twice within i would think it's time complexity is O(2^n). If this recursion is called only once with n-1, i would say it's O(n) - linear and when the function is called within second time with n-2 is also called linear (actually called order of n/2 times. And, O(n/2) = O(n).
But because it's called twice combined, the answer would be "exponential". Please let me know if this is correct.